qemu/tests/hv_avic.py

```diff
@@ -13,6 +14,27 @@ def run(test, params, env):
     param params: the test params
     param env: the test env object
     """
+
+
+    def _get_rhel_main_ver():
+        """
+        Get host main version.
+        """
+        cmd = "cat /etc/redhat-release | awk '{print $6}' | awk -F'.' '{print $1}'"
```

Comment:
Suggest using the `/etc/os-release` instead,  it is a standard file used by many Linux distributions.